library/src/scripts/messages/messageStyles.ts

```diff
@@ -95,6 +95,24 @@ export const messagesClasses = useThemeCache(() => {
         zIndex: 20,
     });
 
+    const innerWrapper = style("innerWrapper", {
+        $nest: {
+            "&&": {
+                flexDirection: "unset",
```

Comment:
Was there a concrete reason for going w/ "unset" here, instead of "initial" or "row"?

`unset`'s behaviour varies based on the parent definition. Do we actually want that?

Reply:
removed  flexDirection: "unset" to innerWrapper